src/views/login/security-question.html

```diff
@@ -33,39 +35,30 @@
 
             <!-- SECURITY ANSWER FIELDS -->
             <div ng-if="!security.loading && !security.invalidCode && security.isThereSelectedHospital()" class="container security-question--container">
-                <form name="formPassword" novalidate>
-                    <div class="form-group">
-                        <label name="text">{{security.Question}}</label>
-                        <div class="security-question--answer-box">
-                            <password-input>
-                                <input type="password"
-                                       id="security_answer_input"
-                                       autocapitalize="none"
-                                       autocorrect="off"
-                                       autocomplete="new-password"
```

Comment:
These removed attributes are still good for controlling how the field is used; I'd keep them (autocapitalize, autocorrect, autocomplete).